source/include/cephadm.rst

```diff
@@ -0,0 +1,82 @@
+cephadm configuration location
+==============================
+
+In kayobe-config repository, under ``etc/kayobe/cephadm.yml`` (or in a specific Kayobe environment
+when using multiple environment, e.g. ``etc/kayobe/environments/production/cephadm.yml``)
+
+StackHPC's cephadm Ansible collection relies on multiple inventory groups:
+- ``mons``
+- ``mgrs``
+- ``osds``
+- ``rgws`` (optional)
+
+Those groups are usually defined in ``etc/kayobe/inventory/groups``.
+
+Running cephadm playbooks
+=========================
+
+In kayobe-config repository, under ``etc/kayobe/ansible`` there is a set of cephadm based playbooks
+utilising stackhpc.cephadm Ansible Galaxy collection.
+
+- ``cephadm.yml`` - runs the end to end process starting with deployment and defining EC profiles/crush rules/pools and users
+- ``cephadm-crush-rules.yml`` - defines Ceph crush rules according
+- ``cephadm-deploy.yml`` - runs the bootstrap/deploy playbook without the additional playbooks
+- ``cephadm-ec-profiles.yml`` - defines Ceph EC profiles
+- ``cephadm-gather-keys.yml`` - gather Ceph configuration and keys and populate kayobe-config
+- ``cephadm-keys.yml`` - defines Ceph users/keys
+- ``cephadm-pools.yml`` - defines Ceph pools\
+
+Running Ceph commands
+=====================
+
+Ceph commands can be run via ``cephadm shell`` utility container:
+
+.. code-block:: console
+   ceph# cephadm shell
+
+This is only possible on the nodes with _admin Ceph orch label set (cephadm will write the admin keys only to those nodes).
+
+Adding a new storage node
+=========================
+
+Add a node to a respective group (e.g. mons) and run ``cephadm-deploy.yml`` playbook.
```

Comment:
Works for OSD nodes, but not others - the spec is only applied during bootstrap.

https://github.com/stackhpc/ansible-collection-cephadm/pull/18/files tried to fix that but got stale.

Reply:
Might be considered as a feature? Spec is normally applied only for OSDs, pools, keys & profiles while a conscious user would specify `cephadm_bootstrap=True` flag to make changes to the deamons. This would need documenting

Reply:
e.g. mons is a bad example

Reply:
Done